coursebook/week-2/research-afternoon.md

```diff
@@ -22,21 +22,25 @@
   + site and user are always separated by a snail/small duck/AT symbol (@);
   + site is always done with at least 3 alphabetical characters, a dot and at least 2 more alphabetical characters
 
-## Accessibility
-1. What does web accessibility mean and why is it important?
-2. What is semantic HTML? Why should we avoid the use of divs? What tag should appear on every page?
-2. What are some basic accessibility checks that can be included in all of our projects? 
-3. Research how to:
-  + write an accessible navbar
-  + write an accessible modal  
-  (Hint: think about semantic HTML, managing focus, adding key handlers, adding aria attributes)
-4. Refactor one of your team's websites (or one from the rest of your cohort if necessary) to reflect what you've learned
+## Git
+1. What is origin ? _eg. `git push origin master`_
+2. How can you find the current origin ? and How can you change the origin's url ?
+3. How to merge between 2 branches using `git merge` ? 
+  What would happen if you do `git merge master current-branch` 
+4. different uses of `git-reset`.
+5. Talk about and give a demo of the following commands, when would you use each one?:
+    - `git stash` & `git stsah pop`
+    - `git diff`
+    - `git log`
+    - `git fetch` 
 
-## CSS
-1. What is responsive design? How does it compare to adaptive design?
-2. Compare the pros and cons of a desktop-first versus mobile-first approach to design. What do you think is generally preferable?
-3. What is BEM CSS and why does it matter?
-4. Compare the % and vw/vh units. What are the pros and cons of each? Where might you use each?
-5. Compare ems and rems as font size units. Why are they used and what would your advice be to someone choosing between the two?
-6. Refactor one of your team's websites to reflect what you've learned
+## Advanced JavaScript
+1. Value vs. reference types in JavaScript
+2. `==` vs. `===`
+3. Scope
+    - What are the different types of scopes 
+    - How does the scope affect variables and variable decleration
+4. The `this` keyword
```

Comment:
i don't think this is a good idea, the `this` keyword is useless unless youre using classes, cause it's considered bad practice to use it in `function` or an object

Reply:
This relates to the scope as well but I can remove because it can be really confusing